flowzone.yml

```diff
@@ -2367,6 +2392,42 @@ jobs:
           path: ${{ runner.temp }}/docs.tar.zst
           retention-days: 90
 
+  npm_sbom:
+    name: Generate SBOM for NPM
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    needs:
+      - is_npm
+      - npm_test
+      - versioned_source
+    if: ${{ needs.is_npm.outputs.npm == 'true' && needs.is_npm.outputs.npm_sbom == 'true' }}
```

Comment:
I wish jobs could build the chain of needed dependencies, so you wouldn't need to explicitly mark `is_npm` as needed, but alas it's required in order to call `needs.is_npm`. LGTM.